packages/react-examples/package.json

```diff
@@ -42,6 +42,31 @@
     "@ctrl/tinycolor": "^3.3.4",
     "@fluentui/azure-themes": "^8.0.5",
     "@fluentui/date-time-utilities": "^8.0.1",
+    "@fluentui/example-data": "^8.1.0",
+    "@fluentui/font-icons-mdl2": "^8.0.1",
+    "@fluentui/foundation-legacy": "^8.0.1",
+    "@fluentui/merge-styles": "^8.0.1",
+    "@fluentui/react": "^8.1.2",
+    "@fluentui/react-avatar": "^9.0.0-alpha.5",
+    "@fluentui/react-badge": "^9.0.0-alpha.5",
+    "@fluentui/react-button": "^9.0.0-alpha.5",
+    "@fluentui/react-cards": "^1.0.0-beta.64",
+    "@fluentui/react-charting": "^5.0.3",
+    "@fluentui/react-checkbox": "^1.0.0-beta.48",
+    "@fluentui/react-components": "^9.0.0-alpha.5",
+    "@fluentui/react-divider": "^9.0.0-alpha.5",
+    "@fluentui/react-docsite-components": "^8.0.3",
+    "@fluentui/react-experiments": "^8.0.3",
+    "@fluentui/react-file-type-icons": "^8.0.1",
+    "@fluentui/react-flex": "^0.2.8",
+    "@fluentui/react-focus": "^8.0.1",
+    "@fluentui/react-hooks": "^8.0.1",
+    "@fluentui/react-icons-mdl2": "^1.0.1",
+    "@fluentui/react-image": "^9.0.0-alpha.5",
+    "@fluentui/react-link": "^9.0.0-alpha.5",
+    "@fluentui/react-make-styles": "^9.0.0-alpha.5",
+    "@fluentui/react-menu": "^0.6.1",
+    "@fluentui/react-provider": "^9.0.0-alpha.5",
```

Comment:
@joschemd Can you please check? These changes look like a merge conflict

Reply:
Updated